src/Controller/ProfileController.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\User;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+#[Route('/profile', name: 'profile_')]
+class ProfileController extends AbstractController
+{
+    #[Route('/{id}', name: 'show')]
+    public function show(User $user): Response
+    {
+        return $this->render('profile/show.html.twig', [
+            'user' => $user,
+        ]);
+    }
```

Comment:
should be:

    #[Route('/profile', name: 'profile')]
    public function show(): Response
    {
        return $this->render('profile/show.html.twig', [
            'user' => $this->getUser(),
        ]);
    }

no?

Reply:
This feature is for future application where you want to see the profile of another person of your community.

Reply:
that's the purpose of UserController::show, isn't it?

Reply:
We want to separate user informations (which include password and other things) and profile which describe the page seen by the user.